ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml

```diff
@@ -1,8 +1,16 @@
 base_images:
+  base:
+    name: "4.5"
+    namespace: ocp
+    tag: base
   dev-scripts:
     name: test
     namespace: openshift-kni
     tag: dev-scripts
+  upi-installer:
+    name: "4.6"
```

Comment:
intentionally 4.6?

Reply:
i thought it needed to be "4.6" but looking through other places that may not be the case... will update